johnwayodi/sm-server
# #161633633 Refactor tests and configuration to accommodate updated endpoints

#### What does this PR do?
Update tests and test data to include more tests for the updated endpoint functions.
#### Description of Task to be completed?
1. Add test data for testing of category section
2. Add tests for the category endpoints.
3. Update all tests accordingly to fix failing builds
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/sm-api-v2.git`
2. Ensure you are in the cloned repo, **sm-api-v2** by default.
3. Create a virtual environment using virtualenv and _pip install -r requirements.txt_
4. `git checkout -b ch-update-tests-161633633 origin/ch-update-tests-161633633`
5. Run **_pytest_** in terminal to run the tests on the application.
#### Any background context you want to provide?
The tests are refactored to include tests for added functionalities which were initially not covered.
#### What are the relevant pivotal tracker stories?
[#161633633](https://www.pivotaltracker.com/story/show/161633633)
#### Screenshots (if appropriate)
![test](https://user-images.githubusercontent.com/10683903/47893507-4f1efd80-de6e-11e8-9938-74b642285498.png)
#### Questions:
None for this section
